doc/source/Interface/ControlsAndShortcuts/index.md

```diff
@@ -0,0 +1,171 @@
+# Controls and Shortcuts #
+
+The following is a list of input device controls and shortcuts for manipulating Gaffer's _Node Graph_, _Viewer_, and _Script Editor_ tabs.
+
+
+## _Node Graph_ Tab ##
+
+> Note :
+> For the following controls and shortcuts, the cursor must hover inside the _Node Graph_ tab.
+
+### Node Creation ###
+
+```eval_rst
+===================================== =============================================
+Action			                      Shortcut or Gesture
+===================================== =============================================
+Show node menu                        Right-click
+"                                     :kbd:`Tab`
+Insert `Dot` at connection            Right-click on connection > `Insert Dot`
+===================================== =============================================
+```
+
+### Node Selection ###
+
+```eval_rst
+===================================== =============================================
+Action			                      Shortcut or Gesture
+===================================== =============================================
+Select all                            :kbd:`Ctrl`-:kbd:`A`
+Clear selection                       :kbd:`Ctrl`-:kbd:`Shift`-:kbd:`A`
+Select node                           Click
+Add/remove node from selection        :kbd:`Shift`-click
+Select nodes                          Click and drag marquee
+Add nodes                             :kbd:`Shift`-click and drag
+Select upstream nodes                 :kbd:`Shift`-:kbd:`Alt`-click
+Select downstream nodes               :kbd:`Shift`-:kbd:`Ctrl`-click
+===================================== =============================================
+```
+
+
+### Node Copying and Deletion ###
+
+```eval_rst
+===================================== =============================================
+Action			                      Shortcut or Gesture
+===================================== =============================================
+Cut node(s)                           :kbd:`Ctrl`-:kbd:`X`
+Copy node(s)                          :kbd:`Ctrl`-:kbd:`C`
+Paste node(s)                         :kbd:`Ctrl`-:kbd:`V`
+Delete node(s)                        :kbd:`Backspace`
+"                                     :kbd:`Delete`
+Enable/disable node(s)                :kbd:`D`
+===================================== =============================================
+```
+
+
+### Connections and Layout ###
+
+```eval_rst
+===================================== =============================================
+Action			                      Shortcut or Gesture
+===================================== =============================================
+Connect plug                          Click and drag plug to another plug
+Disconnect plug                       Click and drag connection to background
+Connect to bookmarked node            Right-click plug > `Connect Bookmark` > select node
```

Comment:
Interesting that we describe connecting to a bookmark before bookmarking itself. Maybe node bookmarks deserves their own section? Or maybe just bringing Navigation ahead of Connections?